src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -7,29 +7,84 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-A sidechain is a separate blockchain which runs in parallel to Ethereum Mainnet and operates independently. It has its own [consensus algorithm](/developers/docs/consensus-mechanisms/) (e.g. [proof-of-authority](https://wikipedia.org/wiki/Proof_of_authority), [Delegated proof-of-stake](https://en.bitcoinwiki.org/wiki/DPoS), [Byzantine fault tolerance](https://decrypt.co/resources/byzantine-fault-tolerance-what-is-it-explained)). It is connected to Mainnet by a two-way bridge.
+A sidechain is a separate blockchain which runs in parallel to Ethereum and is connected to Mainnet by a two-way bridge. Sidechains have separate block parameters and [consensus algorithms](/developers/docs/consensus-mechanisms/), which are often designed for efficient processing of transactions. Using a sidechain involves trade-offs, though, as they do not inherit Ethereum's security properties, unlike [layer 2 scaling solutions](/layer-2/). 
```

Comment:
```suggestion
A sidechain is a separate blockchain that runs independent of Ethereum and is connected to Ethereum Mainnet by a two-way bridge. Sidechains can have separate block parameters and [consensus algorithms](/developers/docs/consensus-mechanisms/), which are often designed for efficient processing of transactions. Using a sidechain involves trade-offs, though, as they do not inherit Ethereum's security properties. Unlike [layer 2 scaling solutions](/layer-2/), sidechains do not post state changes and transaction data back to Ethereum Mainnet.
```

Changed to run independently to give a better mental model that these are in fact not connected to Ethereum in any way other than a bridge. I felt like saying they run in parallel gives the impression that they share more data that a bridge. Added a bit more information to the last sentance about what it means to inherit Ethereum Mainnet security.